test/acceptance/test/add_tests.go

```diff
@@ -7,6 +7,7 @@ package acceptance
 import (
 	"fmt"
 	"os"
+	"os/exec"
```

Comment:
I did not create this file in this PR?

Reply:
All of the files end with `tests`, we could open a separate issue to update all of them if we feel that strongly about it.  I wouldn't just change the one and since it is more than one I wouldn't do it in this PR.